### PR TITLE
Add vmargs setting for qute-ls, disable JVM logging

### LINKED ISSUE #548: Disable JVM logging by default for qute-ls
JVM logging sometimes outputs to standard out. This breaks the language support coming from the language server, because the language client and server communicate through standard in/standard out. Because of this, JVM logging should be disabled by default, and should only be enabled if the user explicitly enables it.

See https://github.com/redhat-developer/vscode-java/issues/2577

### PR DESCRIPTION
**Pull request by datho7561**
Closes #548

Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

File: src/qute/languageServer/quteServerStarter.ts [hunk 1]
```diff
@@ -30,13 +30,19 @@ function prepareParams(): string[] {
     }
   }
 
-  const vmargs = workspace.getConfiguration("quarkus.tools").get("server.vmargs", '');
```
Comment (rgrunber) [inline src/qute/languageServer/quteServerStarter.ts:33]: Just to confirm, this basically did nothing right ? Unless a user actually defined the property manually, which we didn't really "broadcast" via package.json ..

File: package.json [hunk 2]
```diff
@@ -225,6 +225,12 @@
           "description": "Action performed when detected Quarkus properties have an incorrect language.",
           "scope": "window"
         },
+        "qute.server.vmargs": {
+          "type": "string",
+          "scope": "machine",
+          "markdownDescription": "The vmargs to use when launching the Qute language server",
+          "default": "-Xmx100M -XX:+UseG1GC -XX:+UseStringDeduplication"
```
Comment (rgrunber) [inline package.json]: Any reason not to add `-Xlog:disable` here as the default ?

File: package.json [hunk 3]
```diff
@@ -225,6 +225,12 @@
           "description": "Action performed when detected Quarkus properties have an incorrect language.",
           "scope": "window"
         },
+        "qute.server.vmargs": {
```
Comment (rgrunber) [inline package.json:228]: Also, don't forget to update https://github.com/redhat-developer/vscode-quarkus/blob/master/docs/qute/README.md#settings .

File: src/qute/languageServer/quteServerStarter.ts (hunk 1 above)
Comment (datho7561) [reply src/qute/languageServer/quteServerStarter.ts:33]: Yes, this did nothing.

File: package.json (hunk 2 above)
Comment (datho7561) [reply package.json]: I'll add it in

**Pull request merged**